.eslintrc.js

```diff
@@ -207,6 +207,63 @@ module.exports = {
             ignoreRestSiblings: true,
           },
         ],
+        "@typescript-eslint/naming-convention": [
```

Comment:
This will be removed once https://github.com/MetaMask/eslint-config/pull/388 is added to new release and we update the `@metamask/eslint-config` package to 14.x.0 version